install-worker.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+check_ansible_installation() {
+  ansible --version
+  if [[ "$?" != 0 ]]; then
+    sudo apt-get install ansible
+  fi
+}
+
+check_ansible_installation
+
+HOSTS_CONF_FILE="$(pwd)/conf-files/hosts.conf"
+ANSIBLE_HOSTS_FILE="$(pwd)/ansible/worker/hosts"
+ANSIBLE_CFG_FILE="$(pwd)/ansible/worker/ansible.cfg"
+
+# setup hosts
+WORKER_HOST_IP=$(grep ^worker_host $HOSTS_CONF_FILE | awk -F "=" '{print $2}')
+PATTERN_HELPER="\[worker-machine\]"
+WORKER_HOST_IP_PATTERN=$(grep -A1 $PATTERN_HELPER $ANSIBLE_HOSTS_FILE | tail -n 1)
+echo $WORKER_HOST_IP_PATTERN
+sed -i "s/$WORKER_HOST_IP_PATTERN/$WORKER_HOST_IP/" $ANSIBLE_HOSTS_FILE
+
+WORKER_NODE_HOST_IP=$(grep ^worker_node_host $HOSTS_CONF_FILE | awk -F "=" '{print $2}')
+PATTERN_HELPER="\[worker-node-machine\]"
+WORKER_NODE_HOST_IP_PATTERN=$(grep -A1 $PATTERN_HELPER $ANSIBLE_HOSTS_FILE | tail -n 1)
+sed -i "s/$WORKER_NODE_HOST_IP_PATTERN/$WORKER_NODE_HOST_IP/" $ANSIBLE_HOSTS_FILE
```

Comment:
You can use `sed -i '/\[worker-node-machine\]/,/\worker-node-machine:vars\]/{//!d}' ${ANSIBLE_HOSTS_FILE}` to delete everything between [worker-node-machine] and [worker-node-machine:vars].